avocado/plugins/multiplex.py

```diff
@@ -48,7 +47,8 @@ def configure(self, parser):
 
         parser.add_argument('--filter-out', nargs='*', default=[],
                             help='Filter out path(s) from multiplexing')
-        parser.add_argument('--system-wide', action='store_true',
+        parser.add_argument('--system-wide', action='store_false',
```

Comment:
I noticed this is also new on this version of the PR.  On the previous one, you attempted to remove the `--system-wide` arg completely, right?